packages/amplify_api/example/integration_test/rest_tests.dart

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+import 'dart:convert';
+
+import 'package:amplify_api/amplify_api.dart';
+import 'package:amplify_api_example/amplifyconfiguration.dart';
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:amplify_flutter/amplify.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  const path = '/items';
+  const restOptions = RestOptions(
+    path: path,
+  );
+  final restOptionsWithBody =
+      RestOptions(path: path, body: ascii.encode('{"name":"Mow the lawn"}'));
+
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group('REST', () {
+    setUpAll(() async {
+      if (!Amplify.isConfigured) {
```

Comment:
Would remove this - probably want the tests to fail if configure doesn't execute.

Reply:
Want to keep to support running the file individually (rather than from main test file).

Reply:
Just meant to remove the if condition - for example if it's already configured then the first test will fail but maybe not for the reasons we expect. We'd rather have a failure in the setup method if there are issues configuring.

Reply:
discussed offline, keeping as-is here